SQA_AT/src/main/java/model/UserBuilder.java

```diff
@@ -90,8 +90,8 @@ public User constructRandomValidUser() {
                 .build();
```

Comment:
pls check the  video with builder design pattern and pls correct this one
https://www.youtube.com/watch?v=DX1HTfdA2Jo&ab_channel=%D0%A3%D1%80%D0%BE%D0%BA%D0%B8Java